corehq/apps/app_manager/suite_xml.py

```diff
@@ -1416,6 +1416,22 @@ def add_referenced_instances(entry, details_by_id, relevance_by_menu, menu_by_co
 
         entry.require_instance(*instances)
 
+    def get_location_autoselect(self):
```

Comment:
This is basically the same code as here: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/suite_xml.py#L1682-1688

Might make sense to make the function usable in both places (`def get_userdata_autoselect` or something)